docs/reference/api/storage/LittleFileSystem.md

```diff
@@ -12,181 +11,112 @@ for embedded systems, specifically microcontrollers using flash storage.
-lack of a user interface for recovery. With a filesystem that is not resilient
-to power-loss, you rely on luck to not end up with a corrupted filesystem.
-The combination of persistent storage and unpredictable power-loss creates
-difficult to notice bugs that will ruin the user-experience of unlucky users.
+Embedded systems are usually without a shutdown routine and the notable lack of a user interface for recovery. With a file system that is not resilient to power loss, you rely on luck to not end up with a corrupted file system. The combination of persistent storage and unpredictable power loss creates bugs that are difficult to notice and that ruin the experience of unlucky users.
 
-The littlefs is built from the ground up with a power-loss resilient structure
-that uses checksums to limit the assumptions of how the physical storage reacts
-under power-loss. The littlefs provides strong copy-on-write guaruntees and
-always keeps the storage on disk in a valid state.
+The LittleFileSystem has a structure that is resilient to power loss. It uses checksums to limit the assumptions of how the physical storage reacts to power loss. The LittleFileSystem provides copy-on-write guarantees and keeps the storage on disk in a valid state.
 
 ### Wear leveling
 
-Flash storage presents its own unique challenge: Wear. Flash is a
-destructive form of storage, and continuously rewriting data to a block
-will cause that block to wear out and become unwritable. Filesystems that
-don't take wear into account can quickly burn through the blocks were
-frequently updated metadata is stored and result in the premature death
-of the system.
-
-The littlefs is built with wear in mind and the underlying structure
-of the filesystem reactively mutates as the underlying storage develops
-errors over its lifetime. This results in a form of dynamic wear-leveling
-that extends the lifetime of the physical storage proportionally to the
-size of storage. With littlefs, the lifetime of storage can always be extended
-by increasing the size of storage, which is a much cheaper than upgrading the
-erase cycles on storage.
+Flash storage presents its own challenge: wear. Flash is a destructive form of storage, and continuously rewriting data to a block causes that block to wear out and become unwritable. File systems that don't account for wear can quickly burn through the blocks that store frequently updated metadata and result in the premature death of the system.
+
+We accounted for wear when we built the LittleFileSyste, and the underlying structure of the file system reactively mutates as the underlying storage develops errors throughout its lifetime. This results in a form of dynamic wear-leveling that extends the lifetime of the physical storage proportionally to the size of storage. With the LittleFileSystem, you can extend the lifetime of storage by increasing the size of storage, which is cheaper than upgrading the erase cycles on storage.
```

Comment:
You know when you read what you wrote the previous day, and you just get confused by your own wording? Yeah, that happened here for me:
``` diff
- which is cheaper than upgrading the erase cycles on storage
+ which is cheaper than upgrading the storage's erase cycles
```